src/index.js

```diff
@@ -0,0 +1,213 @@
+// LOCATION
+
+currentCity = document.getElementById('cityName').value;
+
+const updateCity = () => {
+  let inputCity = document.getElementById('cityName').value;
+  let showCity = document.getElementById('setCity');
+  showCity.innerHTML = inputCity;
+  currentCity = inputCity;
+};
+
+const resetCity = () => {
+  let cityInput = document.getElementById('cityName');
+  cityInput.value = 'Atlanta';
+  updateCity();
+};
+
+// CONDITIONS
+
+let selectConditions = document.querySelectorAll('input[type="radio"]');
```

Comment:
Nice job using this radio, I'll allow it even though there's an issue with wave 05 (selecting the sky), the requirements specify that a user should select from a dropdown element.